packages/components/socialFeed/NewsFeed/NewsFeedInput.tsx

```diff
@@ -145,6 +146,12 @@ export const NewsFeedInput = React.forwardRef<
     const [isUploadLoading, setIsUploadLoading] = useState(false);
     const [isProgressBarShown, setIsProgressBarShown] = useState(false);
     const [premium, setPremium] = useState(false);
+    const [isMapModal, setMapModal] = useState(false);
+    const [locationSelected, setLocationSelected] = useState<[number, number]>([
+      48.8566, 2.3522,
```

Comment:
how about defaulting to the current user location?
I know we need to ask for permission both in the web and on mobile, but I think it will be a nice UX improvement.
What do you think?

Reply:
![Screenshot 2024-03-12 at 5 03 45 PM](https://github.com/TERITORI/teritori-dapp/assets/154412187/0c2dc380-5a40-4abb-85bb-43c0d521e42d)

in web default permission UI looks like this.

Reply:
@omniwired we don't want to put location, the map is made to localise conceptually a content and "colonize" a map with friends, who choose the position they want, but we need to avoid geolocation in a wallet, for security reasons.

Reply:
Okay so @VishalRakholiya-iView made the modification to don't ask it anymore